src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -261,6 +320,11 @@ public bool ProgramIsRunning()
                     return false;
                 }
             }
+
+            private bool NamedPipeIsBound()
```

Comment:
Is this one line worth the additional method call that we can't inline?

Reply:
I liked the readability here. In the usual circumstance, this code is called once every 10 seconds.
If you think it's important to combine them into one method I can.

Reply:
Nope, it's good. Thanks for the context.